acceptance/tests/resource/mount/mounted.rb

```diff
@@ -2,46 +2,62 @@
 
 confine :except, :platform => ['windows']
 confine :except, :platform => /osx/ # See PUP-4823
-confine :except, :platform => /solaris/
-confine :except, :platform => /aix/
+confine :except, :platform => /solaris/ # See PUP-5201
+
+require 'puppet/acceptance/mount_utils'
+extend Puppet::Acceptance::MountUtils
 
-fstab = '/etc/fstab'
 name = "pl#{rand(999999).to_i}"
 
 agents.each do |agent|
+  fs_file = filesystem_file(agent)
+  fs_type = filesystem_type(agent)
+
   teardown do
-    #(teardown) restore the fstab file
-    on(agent, "mv /tmp/fstab #{fstab}", :acceptable_exit_codes => (0..254))
     #(teardown) umount disk image
     on(agent, "umount /#{name}", :acceptable_exit_codes => (0..254))
     #(teardown) delete disk image
-    on(agent, "rm /tmp/#{name}", :acceptable_exit_codes => (0..254))
+    if agent['platform'] =~ /aix/
+      on(agent, "yes | rmlv #{name}", :acceptable_exit_codes => (0..254))
```

Comment:
@branan see https://github.com/puppetlabs/puppet/pull/4379/files#diff-fbfe185267e39ea41040cf8aa5ef7414R69. We're creating a logical volume for the test, and this is removing it when we're done.